libs/ledger-live-common/src/hw/signMessage/index.ts

```diff
@@ -28,8 +28,8 @@ export const prepareMessageToSign = (
     currency: currency,
     path: freshAddressPath,
     derivationMode: derivationMode as DerivationMode,
-    message: message,
-    rawMessage: "0x" + Buffer.from(message).toString("hex"),
+    message: Buffer.from(message, "hex").toString(),
+    rawMessage: "0x" + message,
```

Comment:
Isn't that a breaking change for the platform sdk since `message` needs to be `hex` now ?
I suppose that we only have the dApp-browser using the `signMessage` so it limits the risks
If we do this we probably need to change the version used in `setPlatformVersion` for LLD at least, no ?

Reply:
So we handle it in the client sdk https://github.com/LedgerHQ/live-app-sdk/pull/118/files/3d2e8cd4b1fbe3db6509090e38bf984bc02b3bb7..f1199155975cca499760d7393c41c8dd5b8777b2#diff-ccf0ddb1fb1db2594f06db30d2c40af22dfc0bf0744a822a71bf4bd8d0880aa5R235
But if a live-app use the old client sdk it will not work correctly, IMO we need to bump the platform version

Reply:
It is breaking a non release function. So not really.
But I agree we can bump the version :)

Reply:
Probably easier to not bump platform to v2 right now

Reply:
Ok, but still, this function has not been released in the SDK...

Reply:
Yeah I'm saying that we don't need to bump if we don't want to